.travis.yml

```diff
@@ -1,10 +1,46 @@
 language: python
 python: '3.5'
-script: "bash ./deploy.sh"
+
+script: echo "no test needed"
+
 env:
   global:
-  - ENCRYPTION_LABEL: "980e848ecaba"
-  - COMMIT_AUTHOR_EMAIL: "loek@d-centralize.nl"
+  - COMMIT_AUTHOR_NAME: ITSLanguage Bot
+  - COMMIT_AUTHOR_EMAIL: its-bot@itslanguage.nl
+  - SITE_DIR: $TRAVIS_BUILD_DIR/.site 
+
 before_install:
   - sudo apt-get -qq update
   - sudo apt-get install -y graphviz
+
+after_success:
+  # Failing will "fail" the build
+  # doing this step in before_deploy
+  # will not.
+  - mkdocs build -d $SITE_DIR
+  # Make sure our script is executable
+  - chmod +x $TRAVIS_BUILD_DIR/gh-pages.sh
```

Comment:
AFAIK, you can't easily pass data from one stage to another with travis, that's why you use the shell script instead of just adding the shell instructions here, right?

Reply:
no, even better, travis has a plugin for this (the ghpages plugin). but that one does not work.. the script has to much stuff to add to this file imo. And, it is the same actually as for the SDK (besides the docs location)... so I am reusing previously obtained knowledge here

Reply:
"that one does not work" -> not completely true: it doesn't do what we want.